doc/release-notes-1.10.rst

```diff
@@ -35,17 +35,32 @@ Kernel
   to have those pointers validated; userspace must never be able to crash the
   kernel by passing it garbage.
 
-
 Architectures
 *************
 
 * nrf52: Add support for LOW_POWER state and SYSTEM_OFF
 * Architecture specific memory domain APIs added
+* Added support for the following ARM SoCs:
+  * NXP i.MX RT
```

Comment:
Needs a blank line before this sublist.  (Can you also add a blank line before the sublist on line 25).

Reply:
This should be `NXP i.MX RT1052` to be more specific (i.MX RT is a family of SoCs)